client/src/App.tsx

```diff
@@ -129,8 +169,9 @@ function App() {
 	};
 
 	const playSound = useCallback(
-		(soundType) => {
+		soundType => {
```

Comment:
this is super long and unnecessary for our purposes. We can write cleaner compact code by doing something like
```audio.current = new Audio[sounds[soundType]]``` which would replace the whole switch statement